include/fiction/algorithms/simulation/sidb/quickexact.hpp

```diff
@@ -495,7 +531,34 @@ class quickexact_impl
         // store the number of SiDBs, since the number of active cells changes during simulation.
         number_of_sidbs = charge_lyt.num_cells();
     }
+
+    /**
+     * This function conducts physical simulation on the given charge layout based on the specified parameters.
+     * If three-state simulation is not required, it performs a two-state simulation with a base number of 2.
+     * If three-state simulation is required, it performs a three-state simulation with a base number of 3.
+     * The simulation results are stored in the 'result' object, including additional simulation parameters.
+     *
+     * @tparam ChargeLyt The type representing the charge layout to simulate.
+     * @param three_state_simulation_required `True` if a three-state simulation is required, `false` otherwise.
+     * @param charge_layout A reference to the charge layout to be simulated.
```

Comment:
```suggestion
     * This function physically simulates the given charge layout based on the specified parameters.
     * If three-state simulation is not required, it performs a two-state simulation with a base number of 2.
     * If three-state simulation is required, it performs a three-state simulation with a base number of 3.
     * The simulation results are stored in the 'result' object, including additional simulation parameters.
     *
     * @tparam ChargeLyt The type representing the charge layout to simulate.
     * @param three_state_simulation_required `true` if a three-state simulation is required, `false` otherwise.
     * @param charge_layout A reference to the charge layout to be simulated.
```